app/templates/direct-award/end-search.html

```diff
@@ -29,49 +29,61 @@
 {% endblock %}
 
 {% block main_content %}
+{% if errors %}
+  <div class="validation-masthead" aria-labelledby="validation-masthead-heading" aria-role="group" tabindex="-1">
+    <p>
+      You need to <a href="#user_understands">confirm that you’ve finished editing your search</a> before you
+      export your results.
+    </p>
+  </div>
+{% endif %}
+
 
 <div class="end-search-page">
   <div class="grid-row">
-    <div class="column-two-thirds">
+    <div class="column-two-thirds dmspeak">
       <header class="page-heading">
         <h1>
-          End your search
+          Before you export your results
         </h1>
       </header>
```

Comment:
I think there is a template for headings now, you can use
```
{% with heading="Before you export your results" %}
  {% include "toolkit/page-heading.html %}
{% endwith %}
```

Reply:
But ewww yuck no :-)